docs/Reference/Pantheon-CLI-Syntax.md

```diff
@@ -608,25 +608,25 @@ rpc-http-cors-origins=["http://medomain.com","https://meotherdomain.com"]
 ```
 
 ```bash tab="Remix IDE domain example"
-# The following allows Remix to interact with your Pantheon node without using MetaMask.
+# The following allows Remix to interact with your Pantheon node.
 
 --rpc-http-cors-origins="http://remix.ethereum.org"
 ```
 
 Specifies domain URLs for CORS validation.
 Domain URLs must be enclosed in double quotes and comma-separated.
 
-Listed domains will be allowed access to node data (whitelisted).
-If your client interacts with Pantheon using a browser app (such as Remix using a direct connection or a block explorer), 
+Listed domains can access to node data (whitelisted).
```

Comment:
This phrase seems to be a bit weird: `Listed domains can access to node data (whitelisted).`

Maybe we can use something like: `Listed domains that can access the node via JSON-RPC`

Reply:
Updated.